test/strategies/kandel/HotKandel.gas.t.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier:	AGPL-3.0
+pragma solidity ^0.8.10;
+
+import "./abstract/CoreKandel.gas.t.sol";
+import {TestToken} from "mgv_test/lib/tokens/TestToken.sol";
+import {Kandel} from "mgv_src/strategies/offer_maker/market_making/kandel/Kandel.sol";
+
+contract HotKandelGasTest is CoreKandelGasTest {
```

Comment:
I thought hotness was reset when test-function is running, so warming up in setup should not have an impact? IF it does have an impact then all storage is hot since everything is deployed in setup? What am I missing?